src/app/views/app_frame/frame/main/side_panel/note_bar/notes/helpers/notes-api.js

```diff
@@ -68,12 +70,23 @@ const deleteNote = async ({ id, accessToken, setNotes, setApiLoading }) => {
 }
 
 const editNote = async ({ id, newContent, accessToken, setNotes, setApiLoading }) => {
-  const response = await axios.put( `https://internmatch-cyrus.gada.io/v7/notes/${id}`, {
-    content: newContent,
-  })
+  setApiLoading( true )
+
+  const response = await axios.put( `https://internmatch-cyrus.gada.io/v7/notes/${id}`,
+    {
+      content: newContent
+    },
+    {
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+      },
+    },
+  )
 
   getAll({ setNotes, accessToken, setApiLoading })
 
+  response.status === 200 ? setApiLoading( false ) : null
```

Comment:
what happens if the status is not 200? We will just get an infinite load? Let's add in some error handling here. Notifying the user that we have had a problem.